addons/web/static/src/js/views/list/list_renderer.js

```diff
@@ -343,6 +391,22 @@ var ListRenderer = BasicRenderer.extend({
             }
         });
     },
+    /**
+     * Classnames "oe_edit_only" and "oe_read_only" aim to only display the cell
+     * in the corresponding mode. This only concerns lists inside form views
+     * (for x2many fields). This function detects the className and stores a
+     * flag on the node's attrs accordingly, to ease further computations.
+     *
+     * @private
+     */
+    _processModeClassNames: function () {
+        const editRegEx = /\boe_edit_only\b/;
```

Comment:
In general, shouldn't we declare static constant variables on top of a file ? I can see that in this case it is more readable because of the variables declared right before they're used, but then why use variables instead of the regexes themselves ? This is more a question for my own information than a remark

Reply:
Premature optimization ^^ I didn't want to create a new regex at each iteration